### DISCUSSION TITLE
Reference/default values

### DISCUSSION THREAD

**Discussion by jasperweiss** [emsesp/EMS-ESP32#1380] [Q&A]
Hi all,

does anyone have some reference/default values for their boiler?
I made an automation in home assistant to control the requested flow temperature, but instead of specifying just the entity, I chose the device as well. Mistakes were made.
Home assistant has been overwriting a whole bunch of values (min/max pump speed, nominal power, dhw flow temperature, etc) and my history log doesn't seem to go back enough to figure out the original values.

**Comment by jasperweiss** [comment]
Luckily I remembered that I have historical data stored in influxDB ^_^
Pro tip: do not select your _device_ as the target in your home assistant automations

**Reply by jasperweiss** [reply]
I still seem to be missing nominal power. Does anyone know what that's supposed to be set to?

**Reply by proddy** [reply]
sorry to hear that. If you can tell us what command you used to set the values we can add a fail safe to prevent the same drama applying to others.

I've done this a few times to, and had to factory reset my boiler (holding down menu & home buttons)

**Reply by jasperweiss** [reply]
Basically I used service `number.set_value` with a target entity as well as a target device. I had a complete brainfart thinking that specifying both made the target more specific so there'd be a smaller chance of accidentally calling it on something else. But of course specifying both will just call the service on both the intended entity as well as the device as a whole.

I only realised something went wrong once the tapwater didn't heat beyond 30 degrees. I initially thought it was due to a write error on the EMS bus but then I noticed that _all_ numeric settings were being updated to the flow temperatures I attempted to set through the automation.

```
service: number.set_value
data: {}
target:
  device_id: 2823b98b71e38588c9b77df22473b5e2
  entity_id: number.boiler_selected_flow_temperature
```

**Reply by MichaelDvP** [reply]
nominal power can not be set in the boiler, it's stored in ems-esp to calculate power. 
In some boilers a fixed setting is in telegram 4 offset 4, if you set nominal power to 0, ems-esp tries to read from the boiler. If it stays zero, there is no setting in the boiler, just insert the value from rating plate.
